src/Diagnostics.jl

```diff
@@ -13,8 +13,18 @@ using ClimaComms
 
 export get_var, init_diagnostics, accumulate_diagnostics!, save_diagnostics, TimeMean
 
+"""
+    AbstractOutputGroup
+
+Abstract type for ClimaCoupler's output diagnostics groups.
```

Comment:
What is an output diagnostic group meant to do?

Reply:
Sure, I added "Each diagnostic group should
contain fields that are of the same type and size, so the extended methods for the group's
operation functions work in the same way for all the fields."